includes/documents/class-wcpdf-invoice.php

```diff
@@ -133,9 +133,16 @@ public function get_sequential_number_store() {
 				// get last year's store if available
 				$last_year         = $now->modify('-1 year');
 				$last_store_name   = $this->get_sequential_number_store_name( $last_year );
-				$last_number_store = new Sequential_Number_Store( $last_store_name, $method );	
+
+				// if the filter 'wpo_wcpdf_document_sequential_number_store' is used and the name is the same
```

Comment:
the year suffix is independent (should be), so the `wpo_wcpdf_document_sequential_number_store` filter will not affect this.